.rubocop_todo.yml

```diff
@@ -714,3 +714,12 @@ Style/TrailingCommaInLiteral:
     - 'Guardfile'
     - 'db/migrate/20140701123203_add_events_per_week_to_conference.rb'
     - 'spec/models/conference_spec.rb'
+
+
+# for ruby 3.0
```

Comment:
I would prefer not to remove the rules, but this should be anyway in `.rubocop.yml`, because this files is automatically generated and this will be removed if we need to regenerate it again :wink:

Reply:
Good point; updating.